Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockAnalyzerInstaller.cs

```diff
@@ -63,11 +62,11 @@ public void AssertExpectedPluginRequested(string key)
     IEnumerable<AnalyzerPlugin> IAnalyzerInstaller.InstallAssemblies(IEnumerable<Plugin> plugins)
     {
         plugins.Should().NotBeNull("Supplied list of plugins should not be null");
-        foreach(var p in plugins)
+        foreach (var p in plugins)
```

Comment:
We generally don't use abbreviations
```suggestion
        foreach (var plugin in plugins)
```